src/globalStyles/global.scss

```diff
@@ -27,11 +27,11 @@ button:active, button:focus, div:focus, div:active {
 
 a {
   cursor:pointer;
-  text-decoration:none !important;
+  text-decoration:none;
```

Comment:
@alisman not sure why these styles are important. Can we remove?

Reply:
this is legitimate.  the style of our links is no underline until hover.   how come you needed to remote !important?  it may not be necessary now that we don't have legacy styles to compete with. but just curious.   we would need to take a look through site to see that things are still consistent.

Reply:
@alisman because `!important` is overwriting the component style when I don't want to have `text-decoration` for the component when hover. I understand you want to standardize the styles, but shouldn't it work already without the `!important`? I cannot overwrite it since it's a global css style on the particular html element.

Reply:
i'm open to taking !important off if it turns out it's not important anymore.  but you should be able to override locally.   you just need to use a more specific selector.  
".someClass a"   will win over "a"  if both have !important

Reply:
I'm not so confident removing it without testing all components. Gonna revert it.